nerdlets/shared/components/cohort-tolerated.js

```diff
@@ -13,8 +13,7 @@ const CohortTolerated = function({
     <>
       <Icon
         className="icon"
-        sizeType={Icon.SIZE_TYPE.NORMAL}
-        type={Icon.TYPE.INTERFACE__STATE__WARNING}
+        type={Icon.TYPE.INTERFACE__STATE__WARNING__SIZE_16}
```

Comment:
same here with the `__SIZE_16`

Reply:
will fix, but I'm curious if it's better for future contributors to have the size value defined? It's seems like an SDK nuance that we default to a particular size, it be odd to see other icon sizes defined, but not for these particular icons.  It's just something to consider if a more verbose approach makes sense for something like this.

Reply:
yeah I could see that in the future! for now as far as I can tell only a handful of the icons have a size suffix (some carets and x's) and adding a suffix to an icon without one listed in the library probably won't render correctly

Reply:
I see. Thanks for explaining!